test/tests/pbs_pp276.py

```diff
@@ -0,0 +1,78 @@
+# coding: utf-8
+
+# Copyright (C) 1994-2016 Altair Engineering, Inc.
+# For more information, contact Altair at www.altair.com.
+# 
+# This file is part of the PBS Professional ("PBS Pro") software.
+#
+# Open Source License Information:
+# 
+# PBS Pro is free software. You can redistribute it and/or modify it under the
+# terms of the GNU Affero General Public License as published by the Free 
+# Software Foundation, either version 3 of the License, or (at your option) any 
+# later version.
+# 
+# PBS Pro is distributed in the hope that it will be useful, but WITHOUT ANY 
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE.  See the GNU Affero General Public License for more details.
+# 
+# You should have received a copy of the GNU Affero General Public License along 
+# with this program.  If not, see <http://www.gnu.org/licenses/>.
+# 
+# Commercial License Information: 
+#
+# The PBS Pro software is licensed under the terms of the GNU Affero General 
+# Public License agreement ("AGPL"), except where a separate commercial license 
+# agreement for PBS Pro version 14 or later has been executed in writing with Altair.
+# 
+# Altair’s dual-license business model allows companies, individuals, and 
+# organizations to create proprietary derivative works of PBS Pro and distribute 
+# them - whether embedded or bundled with other software - under a commercial 
+# license agreement.
+#
+# Use of Altair’s trademarks, including but not limited to "PBS™", 
+# "PBS Professional®", and "PBS Pro™" and Altair’s logos is subject to Altair's 
+# trademark licensing policies.
+
+from ptl.utils.pbs_testsuite import *
+
+
+class TestPp276(PBSTestSuite):
+
+    """
+    This test suite contains hook test to verify that the internal,
+    swig-generated 'pbs_ifl.py' does not cause an exception.
+    """
+    def test_hook(self):
+        """
+        Create a hook, import a hook content that test pbs.server() call.
+        """
+        hook_name = "testhook"
+        hook_body = """
+import pbs
+e = pbs.event()
+s = pbs.server()
+if e.job.id:
+    jid = e.job.id
+else:
+    jid = "newjob"
+pbs.logjobmsg(jid, "server is %s" % (s.name,))
+"""
```

Comment:
Mike, it's because it makes the content of the hook script more readable (when under docstring) to the test writer or whoever else will update the test, rather than reading something like:

hook_body="import pbs\ne=pbs.event()\ns=pbs.server()\n..." and etc...